src/cdi/CPCommon.hh

```diff
@@ -41,6 +41,14 @@ private:
 
 public:
   int32_t get_zaid() const { return zaid; }
+  int32_t get_z() const {
```

Comment:
It might be good to add an explanatory comment or two here, so future draco-ers know you're *intentionally* doing an integer division (and perhaps so they know about the special electron case!)

Reply:
Good point -- done